app/web/features/auth/jail/Jail.tsx

```diff
@@ -80,12 +78,9 @@ export default function Jail() {
           className={classes.section}
         />
       )}
-      {jailInfo?.hasNotSetPassword && (
-        <PasswordSection
-          updateJailed={updateJailed}
-          className={classes.section}
-        />
-      )}
     </>
   );
 }
+{
```

Comment:
We are not using the jail anymore to change the password, since now the use can change the password without even being logged in...

So this page it´s outside the jail system, now this gives me the doubt whether it´s possible that a user end up in a jail system with the attribute `hasNotSetPassword` by accident, I guess this scenario isn´t possible? (just wanted to double check)

Reply:
We shouldn't remove this yet. It's possible for the user to end up here by: not having a password now, then trying to log in. The user would get a login link and end up in jail in the next step.

Reply:
ok, but how? if we implement the new page, this shouldn´t be possible right?
I guess it´s only for the people who already reset its password before this change?

Reply:
the idea to delete this it's to delete dead code (to avoid code which isn't used)

I think that if people click on reset password and they can change it directly on the new page (so in theory never reach this part)

Reply:
It can still leave people in limbo though (they try to log in and get a login link, then it's not obvious why they are jailed and there's no feedback). Let's wait until we clean up the backend to remove password optionality?